common/parsers/index.js

```diff
@@ -0,0 +1,16 @@
+const chrono = require('chrono-node')
+
+const parsers = {
+  date(value) {
+    if (!value) {
+      return value
+    }
+
+    const enGBValue = value
+      .trim()
+      .replace(/^(\d{1,2})([/-])(\d{1,2})(([/-]\d{4}){0,1})$/, '$3$2$1$4')
```

Comment:
Are these changes related to dep upgrades?

Also, my regex brain isn't great, what's the goal of this regex?

Reply:
Thanks for the clarification in slack. As much as I hate them, I wonder if a small comment here might just clarify what we're doing here and why we've had to (because chrono no longer supports en_GB)?

Reply:
Absolutely - just wanted to check this actually worked before testing, documenting and squashing up ;)